compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STNodeTransformer.java

```diff
@@ -285,8 +285,8 @@ public T transform(STRemoteMethodCallActionNode remoteMethodCallActionNode) {
         return transformSyntaxNode(remoteMethodCallActionNode);
     }
 
-    public T transform(STParameterizedTypeDescriptorNode parameterizedTypeDescriptorNode) {
-        return transformSyntaxNode(parameterizedTypeDescriptorNode);
+    public T transform(STMapTypeDescriptorNode mapTypeDescriptorNode) {
+        return transformSyntaxNode(mapTypeDescriptorNode);
```

Comment:
Should we add a simple test to cover this line?

Reply:
BLangNodetranformer is a use case of the external node transformer. Do we any use for the internal node transformer?

Reply:
BLangNodetranformer is a use case of the external node transformer. Do we any use case of the internal node transformer?

Reply:
I think currently we use it only in BallerinaParser.java. Yeh, it's not possible to cover this line without requirement.